### ISSUE TITLE
Use 6.10.3 Node runtime, not 6.10.2

### ISSUE THREAD

**Issue by alulsh** [mapbox/lambda-cfn#54]
I noticed that we're using the Node.js 6.10.2 runtime in the package.json https://github.com/mapbox/lambda-cfn/blob/14015e33f2ca0ec87bffa1866d5f61ab0f9d0ba6/package.json#L7 as well as in the Travis tests https://github.com/mapbox/lambda-cfn/blob/master/.travis.yml#L5.

The AWS Lambda Node.js 6x runtime is 6.10.3 and we say this in the lambda-cfn docs:

> Lambda-cfn (or "Lambda CloudFormation") is a Node.js project that runs on Amazon Web Service's supported Node runtimes, currently Node.js v4.3.2 and v6.10.3.

This is a really minor issue. It's only a minor version difference and I don't see anything critical or important in the [6.10.3 changelog](https://nodejs.org/en/blog/release/v6.10.3/). Also, this is only to ensure consistency when developing locally and testing on Travis so that you're using the same version of Node that's in production. Even though we specified v6.10.2 in the package.json any lambda-cfn code is going to run on 6.10.3 on AWS anyway.

We should update this in lambda-cfn v2.1 or 2.0.1 and also make sure Patrol projects are using 6.10.3 as well.

/cc @zmully

**Comment by zmully** [comment]
I ripped `6.10.2` straight from the docs, so this must be a recent update on Lambda since I did so. Since the Lambda runtime environments are fixed, even though we've specified `6.10.2`, it's going to run whatever version of `6.10.x` they've got in production (now 6.10.3). Although CFN makes this version explicit, the UI does not, which makes me wish they just made the runtime `node6.10` instead so the UI and CFN are the same.

![image](https://user-images.githubusercontent.com/10401798/29375606-9ee764f8-8283-11e7-9ea0-fba357a6d221.png)
